apps/site/src/app/(main)/portal/@applicant/ApplicantPortal.tsx

```diff
@@ -15,6 +15,7 @@ export const enum PortalStatus {
 	waitlisted = "WAITLISTED",
 	waived = "WAIVER_SIGNED",
 	confirmed = "CONFIRMED",
+	attending = "ATTENDING",
 }
 
```

Comment:
Issue: If someone has a status of `ATTENDING`, that means they submitted the waiver. Could you update the `submittedWaiver` variable later in this file to include `PortalStatus.attending`? Otherwise, the `ConfirmAttendance` component will not be shown.